docs/pipeline.rst

```diff
@@ -173,6 +173,18 @@ intent_classifier_sklearn
     to other classifiers it also provides rankings of the labels that did not "win". The spacy intent classifier
     needs to be preceded by a featurizer in the pipeline. This featurizer creates the features used for the classification.
 
+intent_featurizer_regex
+~~~~~~~~~~~~~~~~~~~~~~~
+
+:Short: regex feature creation to support intent and entity classification
+:Outputs: ``text_features`` and ``tokens.pattern``
+:Description:
+    During training, the regex intent featurizer creates a list of `regular expressions` collected from the training data.
```

Comment:
'collected from' is awkward wording I think. Should we just say 'defined in the training data format' ?